snapshot/member_test.go

```diff
@@ -87,7 +90,13 @@ func TestSnapshotV3RestoreMultiMemberAdd(t *testing.T) {
 		t.Fatalf("failed to start the newly added etcd member")
 	}
 
-	cli2, err := clientv3.New(clientv3.Config{Endpoints: []string{newCURLs[0].String()}})
+	eps := make([]string, len(cURLs)+1)
```

Comment:
why do we need to pass all endpoints to the cluster? at this point, we should expect the new member to be ready. if it is, there is an issue in the test.